build_targets

```diff
@@ -2,15 +2,18 @@
 # <device> <build_type> <auto-upload> <upload-recovery>
 beryllium userdebug yes yes
 chef userdebug yes yes
+cheetah userdebug yes yes
```

Comment:
missing cheetah repo?
android_device_google_cheetah else won't find it automatically

Reply:
well won't work that way
on init it'll search on crDroid org for said repo in specific format
if not found, will bail out
https://github.com/crdroidandroid/android_vendor_crdroid/commit/11e962b27f9ce407ed12c3532b42a3080eac3a81